packages/core/src/lib/history.integration.test.ts

```diff
@@ -0,0 +1,94 @@
+import { mkdir, rm, writeFile } from 'node:fs/promises';
+import { join } from 'node:path';
+import { type SimpleGit, simpleGit } from 'simple-git';
+import { afterAll, beforeAll, describe, expect } from 'vitest';
+import { getHashes } from './history';
+
+// we need a separate folder that is not cleaned in `global-setup.ts`, otherwise the tests can't execute in parallel
+const gitTestFolder = 'core-history-git-test';
+describe('git utils in a git repo', () => {
```

Comment:
The only tested function is `getHashes`
```suggestion
describe('getHashes', () => {
```